aiopylgtv/webos_client.py

```diff
@@ -881,11 +893,19 @@ async def set_volume(self, volume):
 
     async def volume_up(self):
         """Volume up."""
-        return await self.request(ep.VOLUME_UP)
+        return await self._volume_step(ep.VOLUME_UP)
 
     async def volume_down(self):
         """Volume down."""
-        return await self.request(ep.VOLUME_DOWN)
+        return await self._volume_step(ep.VOLUME_DOWN)
+
+    async def _volume_step(self, endpoint):
+        """Volume step and conditionally sleep afterwards if a consecutive volume step shouldn't be possible to perform immediately after."""
+        async with self._volume_step_lock:
```

Comment:
Acquire the lock only if the delay is enabled and the sound output is one of the relevant ones.  (Otherwise this could add overhead in the other cases.)